src/node-fallbacks/node-fetch.js

```diff
@@ -1,8 +1,88 @@
-var fetchHandler = globalThis.fetch;
+var { Headers, Request, Response, Blob, File = Blob, FormData, fetch: realFetch } = globalThis;
 
 if ("Bun" in globalThis) {
-  fetchHandler = Bun.fetch;
+  realFetch = Bun.fetch;
 }
 
-export default fetchHandler;
-export { fetchHandler as fetch };
+function fetch(url, opts) {
+  return realFetch(url, opts);
+}
```

Comment:
is there a reason why were wrapping `fetch` like this. before we used to just export the function as is, so node-fetch fetch === fetch

Reply:
`require("node-fetch")` returns the `default` export which means we need to repeat the ESM exports onto it. We don't want to copy that onto the global `fetch` object, so we wrap it. 

This makes it works similar to before node-fetch went esm-only and broke a bunch of packages